docs/content/en/working_with_findings/organizing_engagements_tests/tagging_objects.md

```diff
@@ -0,0 +1,179 @@
+---
+title: "Tags"
+description: "Use Tags to create a new slice of your data model"
+draft: false
+weight: 2
+exclude_search: false
+---
+
+Tags are ideal for grouping objects in a manner that can be filtered out into smaller, more digestible chunks.  They can be used to denote status, or to create custom sets of Product Type, Products, Engagements or Findings across the data model.
+
+In DefectDojo, tags are a first class citizen and are recognized as the facilitators
+of organization within each level of the [data model](../Product_hierarchy).
+
+Here is an example with a Product with two tags and four findings each with a single tag:
+
+![High level example of usage with tags](images/tags-high-level-example.png)
+
+### Tag Formats
+
+Tags can be formatted in any of the following ways:
+- StringWithNoSpaces
+- string-with-hyphens
+- string_with_underscores
+- colons:acceptable
+- "quoted string with spaces"
```

Comment:
```suggestion
```
This one should be removed - this will not longer be supported in the 2.46.0 release